array-methods.js

```diff
@@ -60,19 +107,95 @@ var stateSums = null;
  */
 var sumOfHighInterests = null;
 
+sumOfHighInterests = Object.entries(
+  dataset.bankBalances
```

Comment:
Instead of redoing the calculation from scratch, you could have reused stateSums from above.